deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -300,77 +270,30 @@ def _execute_compaction(
     if params.hash_bucket_count == 1:
         logger.info("Hash bucket count set to 1. Running local merge")
         merge_start = time.monotonic()
-        local_merge_input = generate_local_merge_input(
+        merge_results, total_input_records_count = _run_local_merge(
```

Comment:
Ideally, I would expect this if statement to be moved inside another function `_run_merge`.